openspace.cfg

```diff
@@ -137,12 +137,9 @@ Paths = {
 
 ModuleConfigurations = {
     GlobeBrowsing = {
-        WMSCacheEnabled = false,
-        -- OfflineMode = true,
-        -- NoWarning = true,
-        WMSCacheLocation = "${BASE}/cache_gdal",
-        WMSCacheSize = 1024, -- in megabytes PER DATASET
-        TileCacheSize = 2048 -- for all globes (CPU and GPU memory)
+        TileCacheSize = 2048, -- for all globes (CPU and GPU memory)
+        MRFCacheEnabled = true,
```

Comment:
I don't think we want it to be `true` by default.  Would be good to have the key in here, but defaulted to `false` instead